src/App.jsx

```diff
@@ -0,0 +1,74 @@
+import 'bulma/css/bulma.css';
+import './App.scss';
+import { useState } from 'react';
+
+export const goods = [
+  'Dumplings',
+  'Carrot',
+  'Eggs',
+  'Ice cream',
+  'Apple',
+  'Bread',
+  'Fish',
+  'Honey',
+  'Jam',
+  'Garlic',
+];
+
+export const App = () => {
+  const [selectedGood, setValue] = useState('Jam');
+
+  return (
+    <main className="section container">
+      {selectedGood
+        ? (
+          <h1 className="title is-flex is-align-items-center">
+            {`${selectedGood} is selected`}
+            <button
+              data-cy="ClearButton"
+              type="button"
+              className="delete ml-3"
+              onClick={() => setValue('')}
```

Comment:
You use this logic for the remove button as well. So, move this logic to a separate function to not repeat yourself